lib/northstar-user.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * NorthstarUser.
+ */
+class NorthstarUser {
+
+  // Construct from JSON.
+  constructor(data) {
+    // Computed.
+    this.isAuthorized = false;
+
+    // TODO: object id?
+    this.id = data.id;
+
+    // String.
+    this.firstName = data.first_name;
```

Comment:
Do we want to format the data regardless of what Northstar returns. Capital first letter for first and last name?